packages/smart-contracts/src/lib/artifacts/BatchPayments/index.ts

```diff
@@ -61,10 +61,14 @@ export const batchPaymentsArtifact = new ContractArtifact<BatchPayments>(
           address: '0x0DD57FFe83a53bCbd657e234B16A3e74fEDb8fBA',
           creationBlockNumber: 35498688,
         },
-        zkSyncEra: {
+        zksyncera: {
           address: '0x0C41700ee1B363DB2ebC1a985f65cAf6eC4b1023',
```

Comment:
1. Why is this address identical to the one listed for zkSync Era [`BatchConversionPayments`](https://github.com/RequestNetwork/requestNetwork/pull/1261/files#diff-a7c454cdf39eb8aa49bbdbd7dd4ed16ab0ab8271d6fffb702c06ce687ee57101R68)

2. Why does the [zkSync Era explorer](https://explorer.zksync.io/address/0x0C41700ee1B363DB2ebC1a985f65cAf6eC4b1023#contract) show the contract name "BatchConversionPayments" instead of "BatchPayments"?
![image](https://github.com/RequestNetwork/requestNetwork/assets/2530913/b4fe700b-f466-43a1-8064-794f21c1304c)

Reply:
The BatchConversionPayments is the contract that is used in production, so that's why that one is deployed.

At the time I thought BatchPayments was the one we were using but if we see the block explorer on mainnet: https://etherscan.io/address/0x0dd57ffe83a53bcbd657e234b16a3e74fedb8fba it hasn't been used for more than a year.

So if you want to clean things up, we can just remove the entries from BatchPayments/index.ts for both prod and test zksync.

Reply:
Added issue to fix :+1: https://github.com/RequestNetwork/requestNetwork/issues/1303